### ISSUE TITLE
PNG codec error and image zooming/moving trouble

### ISSUE THREAD

**Issue by ucrainis** [nroduit/Weasis#553]
### Describe the bug. What happened?

Good day,

after updating to 4.4.0 version of the app occured several errors and I cannot work any more: can't load PNG format images - I get every time error saying that app cannot read this image (attached screenshot).
And another one - loading any other image (JPG, BMP etc) I cannot scroll image - it got stucked in the corner (attached second screenshot) without possibility to pan (move on it) image, zoom it in or out and to look resized to best fit version of the image.
![1](https://github.com/nroduit/Weasis/assets/169903081/3653cce0-cee9-490f-bcb5-439a5d30b50f)
![2](https://github.com/nroduit/Weasis/assets/169903081/0428b701-7b57-40ba-ba30-3a5f05717929)

### What version of Weasis are you running?

4.4.0

### On which system the problem occurs?

Windows

**Comment by nroduit** [comment]
I can't reproduce the 2 problems mentioned. Please provide [logs](https://weasis.org/en/tutorials/logging) and/or sample images.

**Comment by ucrainis** [comment]
> I can't reproduce the 2 problems mentioned. Please provide [logs](https://weasis.org/en/tutorials/logging) and/or sample images.

I looked into the logging folder, but I saw that the last logs were before update - since then all those times I opened app and tryed to do something - nothing new adds there. The problem is the same on two different PCs: Windows 7 and Windows 10. If it helps - I can do a video as I try to do the same things mentioned.

**Comment by ucrainis** [comment]
> I can't reproduce the 2 problems mentioned. Please provide [logs](https://weasis.org/en/tutorials/logging) and/or sample images.

I checked preferences: logging was turned off with new update. So I share the log-file after trying to do this things and quitting app.
[default.log](https://github.com/nroduit/Weasis/files/15331954/default.log)

**Comment by nroduit** [comment]
The problem lies in the OpenCV library, with an unexpected error.

Without an example to reproduce the problem, I don't see how I can help.

**Comment by ucrainis** [comment]
> The problem lies in the OpenCV library, with an unexpected error.
> 
> Without an example to reproduce the problem, I don't see how I can help.

Without an example - you mean I need to provide video of what I am doing or something else?

**Comment by nroduit** [comment]
I need a test case where the problem can be reproduced. Please send me an image that cannot be read.

**Comment by ucrainis** [comment]
> I need a test case where the problem can be reproduced. Please send me an image that cannot be read.

Yep, I send the .png file, that gives error with codec.
About the second problem - it has to do simply with every other file that app reads.
![pan](https://github.com/nroduit/Weasis/assets/169903081/9f24c3dd-8904-44af-b98f-7eac21c9b5d0)

**Comment by ucrainis** [comment]
Now I downgraded to 4.3.0 and everything works fine.

**Comment by nroduit** [comment]
I've tested it on several Windows 10 and 11 systems and I can't reproduce the error.

Do you have recent or specific versions of Windows?

See this [post](https://github.com/nroduit/Weasis/issues/431#issuecomment-1623022992) even though normally with recent compile mode Visual C++ redistributable should not be necessary.

**Comment by ucrainis** [comment]
> I've tested it on several Windows 10 and 11 systems and I can't reproduce the error.
> 
> Do you have recent or specific versions of Windows?
> 
> See this [post](https://github.com/nroduit/Weasis/issues/431#issuecomment-1623022992) even though normally with recent compile mode Visual C++ redistributable should not be necessary.

I have PCs with Windows 7 Professional and Windows 10 Home with all actual upgrades. But the bug is version-dependent, not Windows-dependent - 4.3.0 works well on both machines. I do have Visual C++ installed on my computers. 

So, I tryed installing again 4.4.0 (previously installed Visual C++ redistributable 2015 and Universal C Runtime) - nothing has changed at all: both bugs still appear. Then I downgraded again to 4.3.0 - the app works fine again. I guess I will just stay on 4.3.0 release.

**Comment by nroduit** [comment]
I haven't found any other users with this problem.
As the problem is in a third-party library, there are no error messages and no procedure to reproduce the problem, I'm not going to investigate further.

**Comment by ucrainis** [comment]
> Does this problem still exist in version 4.6.5, because I saw that there were many fixes in the png decoder.

No. A minute ago I updated to the 4.6.5, checked and now everything works fine. Thank you!
